### PR TITLE
Changelog v1.1.0

### PR DESCRIPTION
**Pull request by purva-thakre**
## Description
Provide a brief description of the PR's purpose here. If your PR is supposed to fix an existing issue, use
a [keyword](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue) to link your PR to the issue.

## Changes
Notable changes that this PR has either accomplished or will accomplish. Feel free to add more lines to the itemized list
below.

  -  [ ] Change 1

## Checklist
Before marking your PR ready for review, make sure you checked the following locally. If this is your first PR, you might be notified of some workflow failures after a maintainer has approved the workflow jobs to be run on your PR. 

Additional information is available in the [documentation](https://toqito.readthedocs.io/en/latest/contributing.html#testing).

  -  [ ] Use `ruff` for errors related to code style and formatting.
  -  [ ] Verify all previous and newly added unit tests pass in `pytest`.
  -  [ ] Check the documentation build does not lead to any failures. `Sphinx` build can be checked locally for any failures related to your PR
  -  [ ] Use `linkcheck` to check for broken links in the documentation
  -  [ ] Use `doctest` to verify the examples in the function docstrings work as expected.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,174 +1,339 @@
...
+- Tensor products now utilize fast exponentiation, @tnemoz in #809
+- Documentation employs a different theme, search inside documentation is also fixes, @vprusso and @purva-thakre in #836
+
+### New Features
+
+- 
+
+### Merged Pull Requests
+
+- full automated changelog here
```
[the remark this answers is not in the document]
Comment (vprusso) [reply CHANGELOG.md]: Looks great--I like that format!

**Comment by purva-thakre** [comment]
Remember to merge this PR after #853 and #842

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,174 +1,363 @@
...
+- toqito is compatible with Numpy 2.0, @purva-thakre in https://github.com/vprusso/toqito/pull/754
+- Infix operator replaces instances of `*` for matrix multiplication, @vprusso in https://github.com/vprusso/toqito/pull/797
+- `scikit-image` was removed from `setup.py`, @vprusso in https://github.com/vprusso/toqito/pull/819
+- Deprecated usage of `csr_matrix` and `dia_matrix` in `perms/permute_systems` was removed, @purva-thakre in https://github.com/vprusso/toqito/pull/816
+- BCS conversion error for upper and lower bounds on the quantum value was corrected, @vprusso in https://github.com/vprusso/toqito/pull/811
+- Unused `cvx_kron` function was removed, @vprusso in https://github.com/vprusso/toqito/pull/795
+
+### Merged Pull Requests
+
+- full automated changelog here
```
Comment (purva-thakre) [inline CHANGELOG.md:43]: Add the full automated changelog in this subsection.

**Review by vprusso — APPROVED** [review]
LGTM!

**Review by vprusso — APPROVED** [review]
LGTM! Merging in.

**Pull request merged**